client/question/domain/FeedbackObjectFactory.js

```diff
@@ -139,6 +139,17 @@ tie.factory('FeedbackObjectFactory', [
     };
 
     /**
+     * Appends a FeedbackParagraph of type output to the _paragraphs
+     * Array
+     *
+     * @param {string} text String of code to be inserted as code output
```

Comment:
Perhaps `code` might be a better variable name? Or even `outputString`? Unclear to me, from reading the docstring, whether this is student's code or the output printout from Python -- I'd assume the latter, but that isn't really code.

Reply:
outputString SGTM. Will change it to that.